.github/workflows/cmake.yml

```diff
@@ -16,7 +16,7 @@ jobs:
             name: Linux
             container: wpilib/roborio-cross-ubuntu:2023-22.04
             flags: ""
-          - os: macOS-11
+          - os: macOS-12
```

Comment:
Is it necessary to build with 12 or just target 11?

Reply:
I think we have to build on 12 to use Xcode 14. https://developer.apple.com/support/xcode/